### DISCUSSION TITLE
Avis sur la classe Path / module pathlib

### DISCUSSION THREAD

**Discussion by PierreS-alpha** [IPGP/autorino#3] [General]
Resalut,

Pour les définition des chemins, vaut il mieux utiliser la classe Path  du module pathlib, ou bien de simple strings suffisent?
ou les deux?

là aussi, je pense aux utilisateurs néophytes qui devront maintenir les codes en opérationnel

https://www.freecodecamp.org/news/how-to-use-pathlib-module-in-python/

**Comment by PBoissier** [comment]
Salut,
Désolé pour le délai!
Je suis partisan d'utiliser la classe Path. 
Ça permet en particulier d'effectuer toutes les opérations sur le système de fichier en Python pur sans embarquer des appels systèmes. C'est mieux en particulier pour la gestion des exceptions.
Ca permet aussi d'itérer dans les dossier simplement.
